cupy/_core/core.pyx

```diff
@@ -2258,6 +2243,27 @@ cpdef function.Module compile_with_cache(
     if _cuda_path is not None:
         options += ('-I' + os.path.join(_cuda_path, 'include'),)
```

Comment:
Perhaps we should opt out this to avoid caching headers from CTK?

Reply:
Thinking out loud, Jitify wouldn't work without headers from CTK... (ex: `cuda_fp16.h` etc include CTK headers)